camera_manager.py

```diff
@@ -0,0 +1,154 @@
+import subprocess
+from time import sleep
+import cv2
+
+def get_camera_name_and_paths() -> dict[str: str]:
+    # Run the v4l2-ctl command to list available cameras
+    command = "v4l2-ctl --list-devices"
```

Comment:
Parsing `v412-ctl` directly can be dangerous - it's more of a user-facing tool. 

It'd be a good idea to either: 
- use a library to ask Video4Linux directly, or
- stick down a quick and dirty FIXME + log() to remind folks that this can break.

Reply:
Added a todo for later